playbooks/ci-ceph_swift/tasks/pre-deploy.yml

```diff
@@ -35,6 +35,32 @@
     - name: umount ephemeral disk on ceph_osds
       shell: umount /dev/{{ item }}
       with_items: "{{ ceph.disks }}"
+      ignore_errors: true
+
+- name: umount ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: umount ephemeral disk on swiftnode
+      shell: umount /dev/{{ item.disk }}
+      with_items: "{{ swift.disks }}"
+      ignore_errors: true
+
+- name: create new partition table for ephemeral disk on swiftnode
+  hosts: swiftnode
+  tasks:
+    - name: create new msdos partition table for ephemeral disk
+      shell: parted /dev/{{ item.disk }} --script mklabel msdos
+      with_items: "{{ swift.disks }}"
+
+- name: deploy fake lshw file on swiftnode
+  hosts: swiftnode
+  tasks:
```

Comment:
Just want to make sure of something here. This playbook is only ever going to be run once, to prepare nodes for deployment, and then never run again, is that correct?